addons/account/models/account_move.py

```diff
@@ -876,7 +876,7 @@ def _apply_cash_rounding(self, diff_balance, diff_amount_currency, cash_rounding
             if in_draft_mode:
                 cash_rounding_line.update(cash_rounding_line._get_fields_onchange_balance(force_computation=True))
 
-        existing_cash_rounding_line = self.line_ids.filtered(lambda line: line.is_rounding_line)
+        existing_cash_rounding_line = self.line_ids.filtered(lambda line: line.line_type in ['invl_cash_rounding', 'inv_tax_cash_rounding'])
```

Comment:
Don't remove the tax line :scream:

Reply:
I don't, it removes cash rounding lines. Just the two types.